### DISCUSSION TITLE
Problem A: Pagination of Data

### DISCUSSION THREAD

**Discussion by BradKML** [cinemagoer/cinemagoer#338] [Ideas]
Observing
- https://github.com/alberanid/imdbpy/issues/297
- https://github.com/alberanid/imdbpy/issues/324

There has to be a function that:
- Auto-paginates
- Stops at a certain function
  - When X results are out (common)
  - When I see movie Y in the result (less useful)
- Detect if it is already the last page (prevents bot detection and spamming)

**Comment by BradKML** [comment]
@alberanid formalities on this might be good to address.

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
